transport_nantes/asso_tn/utils.py

```diff
@@ -56,8 +70,9 @@ def make_timed_token(string_key, minutes, int_key=0, test_value_now=None):
     int_key_str = int_to_base36(int(int_key))
     hmac = salted_hmac(soon_seconds + int_key_str, rand_value + str(string_key)
                        ).hexdigest()[:20]
-    token = '{rnd}|{e}|{t}|{p}|{h}'.format(
+    token = '{rnd}|{hr}|{e}|{t}|{p}|{h}'.format(
```

Comment:
Adding another element to the token will raise a ValueError when other functions using this token will unpack the resulting token.

Reply:
Isn't the only function to do that `token_validate`?

Reply:
Yes, there's a race condition.  All those pending tokens out there...  ;-)

Reply:
Though that's a reminder that we need to make sure we're always robust against a bad token coming in, offering something reasonably logical to help the user recover.

Reply:
The token_valid function will return a variable number of values, one for each value seperated by a pipe.
And we need to store the values out of token_valid, to extract the email, and send_record_id if any.

Reply:
The job of `token_valid()` is to extract the two values we put in.  No other entity should be trying to parse the token and expecting not to be fragile.

Can you construct a scenario and show me in code how you could call `make_timed_token()` and then pass the token to `valid_token()` and have the string and int that you recover not be what you put in?